code/modules/client/preference_setup/loadout/loadout_factions.dm

```diff
@@ -132,13 +112,13 @@
 
 /datum/gear/faction/eridani_beret
 	display_name = "eridani beret"
```

Comment:
Change this, they're no longer EPMC but PMCG.

Reply:
Unless they're still EPMC but for some reason you changed the type path to say "pmc", in which case, change it back.